lib/vmdb/appliance.rb

```diff
@@ -36,6 +36,14 @@ def self.USER_AGENT
       "#{self.PRODUCT_NAME}/#{self.VERSION}".freeze
     end
 
+    def self.check_automate_disk_version_against_db
+      rh_domain = MiqAeDomain.find_by(:name => "RedHat")
+      if rh_domain.version != rh_domain.available_version
+        startup.info("RedHat domain on disk differs from domain in db:")
```

Comment:
The only place I see `startup` defined is as a variable in `self.log_server_identity`.  Was that going to be turned into a method for reuse?